### PR TITLE
[#175892280] Fix decoder body response 201 and update max logo size

### PR DESCRIPTION
**Pull request by infantesimone**
This PR introduces:
- fix api response decoder for 201, decoding a empty body with `{}` instead `undefined`
- update max logo size to 50 mb

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ secureExpressApp(app);
 
 app.use(cors());
 app.use(cookieParser());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.urlencoded({ extended: true, limit: "50mb" }));
```
Comment (gunzip) [inline src/app.ts:90]: i guess this should be setup into io-functions-admin as well isn't it @AleDore ?

Comment (AleDore) [reply src/app.ts:90]: Yes, but we need another PR on io-functions-admin :)

**Review by AleDore — APPROVED** [review]
LGTM, remember to open another PR on io-functions-admin in order to enable maxLogoSize :)

File: src/app.ts (hunk 1 above)
Comment (gunzip) [reply src/app.ts:90]: is this a resonable limit?

**Comment by AleDore** [comment]
Can we merge this? @balanza @gunzip

**Review by gunzip — APPROVED** [review]

**Pull request merged**